src/Quill.php

```diff
@@ -543,6 +582,10 @@ public function registerClientScript(): void
         }
         $asset->theme = $this->theme;
 
+        if ($this->isSmartBreak() && $this->localAssets) {
```

Comment:
Since this is the third asset that checks `localAssets` property could you refactor all of them to check for it first and then set all assets? Like

```
if ($this->localAssets) {
    if ($this->isHighlightJs()) {
    ...
    }
    if ($this->isKatex()) {
    ....
    }
    if ($this->isSmartBreak()) {
    ....
    }
} else {
    if ($this->isHighlightJs()) {
    ...
    }
    if ($this->isKatex()) {
    ....
    }
}
```